stock_customer_deposit_sale_margin/readme/CONTEXT.md

```diff
@@ -0,0 +1 @@
+This module was developed because cost in sale order line when deliver customer deposit should be zero because purchase price is in sale order that made customer deposit, otherwise the cost would be posted twice.
```

Comment:
suggestion: reword and simplify to make it easier to understand

```suggestion
This module was created to make sure that the cost on a sale order line is zero when delivering from customer deposits. This is needed because the purchase price already accounts for the customer deposit in the sale order. Without this module, the cost would be counted twice in the accounting.
```